packages/core/src/core/__test__/vector-db.test.ts

```
@@ -0,0 +1,357 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    globals: true,
+    environment: "node",
+    coverage: {
+      reporter: ["text", "json", "html"],
+    },
+  },
+});
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)